packages/backend/src/auth/auth.service.ts

```diff
@@ -18,7 +18,7 @@ export class AuthService {
       throw new NotFoundException();
     }
 
-    const updatedUser = await this.prisma.user.update({
+    return await this.prisma.user.update({
```

Comment:
The [immediate-return](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/prefer-immediate-return.md#prefer-immediate-return) rule.

They arg that function name should be enough for caller to understand what he will get from this method